packages/next-server/server/config.js

```diff
@@ -18,7 +18,9 @@ const defaultConfig = {
   onDemandEntries: {
     maxInactiveAge: 60 * 1000,
     pagesBufferLength: 2,
-    websocketPort: 0
+    websocketPort: 0,
+    websocketProxyPath: null,
+    websocketProxyPort: null
```

Comment:
Why have another port next to this one? 🤔

Reply:
I reserve `7001` on `nginx` and `7002` in Next. The two can't be the same port.

Reply:
Ah now I get it, sounds good to me 💯